shell/src/chain_manager.rs

```diff
@@ -1713,7 +1706,7 @@ impl Receive<DisconnectStalledPeers> for ChainManager {
                         Ok(result) => result,
                         Err(_) => {
                             warn!(ctx.system.log(), "Failed to collect current local head";
-                                            "peer_id" => state.peer_id.peer_id_marker.clone(), "peer_ip" => state.peer_id.peer_address.to_string(), "peer" => state.peer_id.peer_ref.name(), "peer_uri" => uri.to_string());
+                                            "peer_ip" => state.peer_id.address.to_string());
```

Comment:
@binier 
I tried to unify logs for peers a little  but address is not enought here, 
peer_id_marker there was a CryptoboxPublicKeyHash.to_base58_check, so I would like to have at least that "idX123456.." with socket_address in logs for later investigations, we needed/used that already several times

Reply:
Hm... You are right, especially in this case we won't be able to identify same peer across multiple connection/disconnections.

Reply:
@binier 
we need to be able identify peer by address and identity from logs,
if dont have time to do this now, we will do it later: https://viablesystems.atlassian.net/browse/TE-683